T/TopologicalDataAnalysis/Deps.toml

```diff
@@ -0,0 +1,2 @@
+[0]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
`Test` is usually a [test-only dependency](https://pkgdocs.julialang.org/v1/creating-packages/#Test-specific-dependencies-in-Julia-1.2-and-above).  Quickly skimming through your package it doesn't look to me you use `Test` there.

[noblock]